### PR TITLE
Super minor changes

### PR DESCRIPTION
**Pull request by ttgmichael**
Started new character, a relook at lines.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -14655,7 +14655,7 @@ ETC_20150918_014656	{memo X}Suspicious Device
 ETC_20150918_014657	Gailryus Legwyn
 ETC_20150918_014658	Vendor
 ETC_20150918_014659	{memo X}Spirit Grass
-ETC_20150918_014660	
+ETC_20150918_014660	Dungeon of the Fallen Legwyn Family Notice Board
```
Comment (Sunaries) [inline ETC.tsv]: Change to: Fallen Legwyn Family Dungeon Notice Board

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -2259,7 +2259,7 @@ QUEST_LV_0100_20150317_002254	I'm furious and upset. {nl} If we got here earlier
 QUEST_LV_0100_20150317_002255	$Dear Revelator, I have a favor to ask of you. {nl}Can you recover my comrades' mementos?
 QUEST_LV_0100_20150317_002256	{memo X}It's bad enough that they were killed by the Vubbes, but even their keepsakes are being abused. How awful...
 QUEST_LV_0100_20150317_002257	{memo X}Thank you. {nl} My comrades will now be able to rest in peace.
-QUEST_LV_0100_20150317_002258	$Oh my God. The Vubbes stole all our food supplies. {nl} They even took our food for today... Please help us, we need your strength.
+QUEST_LV_0100_20150317_002258	$Oh my god. The Vubbes stole all our food supplies. {nl} They even took our food for today... Please help us, we need your strength.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why lowercase?

Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Do you really refer to God when you say this xD? (my God)
If not really, then it's just a figurative god right?

**Comment by ttgmichael** [comment]
This ought to be good to merge; I don't see any other fixes to make in 12 lines. xD

**Comment by Sunaries** [comment]
Good to merge. :)

**Comment by imcgames** [comment]
:+1:

**Pull request merged**